api/test/FakePlatformApis.js

```diff
@@ -0,0 +1,77 @@
+const express = require('express');
+const app = express();
+const port = 8090;
+
+app.get('/api/v1/claimants', (req, res) => {
+  if (!req.headers['X-API-Key'] === 'super-secret-api-key') {
+    res.sendStatus(403);
+  } else {
+    if (req.query.first_name === 'flor' && req.query.last_name === 'beden') {
+      res.send({
+        claimants:
+          [{
+            claimId: 6060591,
+            checkDigit: '3',
```

Comment:
What is a check digit? Been seeing it but never remembered to ask 😅 Is this a column saved in the DB too for each record?

Reply:
Yeah, it forms part of the ID that single view ends up using as well. Tbh I can't remember it's actual meaning.